haiku-apps/simplebackup/simplebackup-2.0.1.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="Graphical interface to run the zip command"
+DESCRIPTION="This application makes it easy to create and update backups of \
+large amounts of data.  For example, you can create a backup of your home \
+directory and run this application regularly to make sure the backup is up to \
+date.  SimpleBackup also accepts command line options, so it is easily \
+scriptable and can be used with scheduling applications."
+HOMEPAGE="https://perelandra0x309.github.io/simplebackup"
+COPYRIGHT="2010-2016 Brian Hill"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/Perelandra0x309/simplebackup/archive/v${portVersion}.tar.gz"
+CHECKSUM_SHA256="463df905df35a74e81127d1c28d20a1f2c94ff1b89f6939abf081ecc70e60c62"
+SOURCE_DIR="simplebackup-${portVersion}"
```

Comment:
In case anyone would like to use the same haikuports tree with all their "flavour" architectures, here's what I have setup:

* In my "data" partition I have 2 symbolic links:
/data//haikuports/packages -> /home/packages
/data//haikuports/repository -> /home/repository
as well as 6 directories:
/data//packages.x86_gcc2
/data//packages.x86
/data//packages.x86_64
/data//repository.x86_gcc2
/data//repository.x86
/data//repository.x86_64

* In my "Haiku-x86_gcc2" bootable partition I have 2 symbolic links:
/boot/home/packages -> /data/packages.x86_gcc2
/boot/home/repository -> /data/repository.x86_gcc2

* In my "Haiku-x86" bootable partition I have 2 symbolic links:
/boot/home/packages -> /data/packages.x86
/boot/home/repository -> /data/repository.x86

* In my "Haiku-x86_64" bootable partition I have 2 symbolic links:
/boot/home/packages -> /data/packages.x86_64
/boot/home/repository -> /data/repository.x86_64

All this allows to have separate packages and repository subdirectories, and yet, share the same haikuports tree. I came to the conclusion that this was the easiest way to have a single haikuports tree and keep things working. During my setup I understood that:
* Mixing the packages directories from the 3 platforms is not a good idea.
* Mixing the repository directories from the 3 platforms is a very bad idea.

All I have to do when I reboot into another architecture is remember to clean the work-* subirectories, especially when I build for primary architectures.

@Perelandra0x309: sorry for posting here what I should put in a wiki...